src/tasks/trigger/AutoSkipDialogTask.py

```
@@ -0,0 +1,43 @@
+
+class AutoSkipDialogTask(BaseNTETask, TriggerTask):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.default_config = {'_enabled': True}
+        self.name = "自动跳过剧情"
+        self.icon = FluentIcon.ACCEPT
+
+    def run(self):
+        if self.find_one(Labels.skip_dialog, horizontal_variance=0.05):
+            self.log_info("检测到跳过对话框，正在自动跳过...")
```

Ruff (0.15.10)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)